Week1/wdl_parsing/test_WdlV1_1.py

```diff
@@ -0,0 +1,159 @@
+f = open("Qc.wdl", "r")
+
+import sys
+from antlr4 import *
+from WdlV1_1Lexer import WdlV1_1Lexer
+from WdlV1_1Parser import WdlV1_1Parser
+from WdlV1_1ParserVisitor import WdlV1_1ParserVisitor
+import cwl_utils.parser_v1_2 as cwl
+
+text = InputStream(f.read())
+lexer = WdlV1_1Lexer(text)
+stream = CommonTokenStream(lexer)
+parser = WdlV1_1Parser(stream)
+tree = parser.document()
+
+#create WdlV1_1ParserVisitor object and return all inputs, outputs, etc
+ast = WdlV1_1ParserVisitor()
+ast = ast.walk_tree(tree)
+
+#get input type and name
+def get_inputs(ast):
+    print('TASK INPUTS')
+    for a in ast.task_inputs:
+        print(a[0], " ", a[1])
+
+#get runtime requirements
+def get_runtime(ast):
+    print('TASK RUNTIME')
+    for a in ast.task_runtime:
+        print(a[0]," ",a[1])
+
+#get output type, name, expression
+def get_outputs(ast):
+    print('TASK OUTPUT')
+    for a in ast.task_outputs:
+        print(a[0]," ",a[1]," ",a[2])
+
+#get command
+def get_command(ast):
+    print('TASK COMMAND')
+    print(ast.task_command)
+
+#get_inputs(ast)
+#get_runtime(ast)
+#get_outputs(ast)
+#get_command(ast)
+
+#returns the entire command including "command{........}"
+command = ast.task_command
+command = command[command.find("{")+1:-1] #removing the command{} part
+command = command.strip().split("\\") #split by '\'
+
+base_command = ""
+command_arguments = []
+
+for a in command:
+    #if it contains = , it's taken as an argument
+    if "=" in a:
+        command_arguments.append(a.strip())
+    #else add to base command
+    else:
+        base_command+=a
+
```

Comment:
This works, good!

1. `=` is not required for a CWL parameter reference or CWL expression. Search instead for `$(` or `${`
2. what happens if a later command argument does not contain a `=` ? Then it would be added to the `baseCommand` and that will affect the command line order: https://www.commonwl.org/v1.2/CommandLineTool.html#Input_binding

Reply:
but this won't work for an argument like 'OQ=true' as mentioned in this wdl file right?